eos/b-decays/b-to-d-l-nu.cc

```diff
@@ -81,21 +92,35 @@ namespace eos
             double fp = form_factors->f_p(s);
             double f0 = form_factors->f_0(s);
             double lam = lambda(m_B * m_B, m_D * m_D, s);
-            double norm = power_of<2>(g_fermi())
-                / (384.0 * power_of<3>(M_PI * m_B));
+            double pBtoD = sqrt(lam)/(2.0 * m_B);
+            double rmls = m_l * m_l / s;
             // make sure we return NaN if s < m_l^2
-            double sqrtv = sqrt(1.0 - m_l * m_l / s);
-            double v = sqrtv * sqrtv, v2 = v * v;
```

Comment:
Please keep the notation using v, since that has a physical meaning (v = lepton velocity in the dilepton rest frame).